src/algebra/curve.rs

```diff
@@ -873,102 +867,85 @@ impl StarkPointResult {
 #[cfg(test)]
 mod test {
     use rand::{thread_rng, RngCore};
-    use starknet_curve::{curve_params::GENERATOR, ProjectivePoint};
 
-    use crate::algebra::test_helper::{
-        arkworks_point_to_starknet, compare_points, prime_field_to_starknet_felt, random_point,
-        starknet_rs_scalar_mul,
-    };
+    use crate::algebra::test_helper::{random_point, TestCurve};
 
     use super::*;
     /// Test that the generators are the same between the two curve representations
     #[test]
     fn test_generators() {
-        let generator_1 = StarkPoint::generator();
-        let generator_2 = ProjectivePoint::from_affine_point(&GENERATOR);
+        // let generator_1 = CurvePoint::generator();
```

Comment:
are these commented-out tests dead code to be removed? or will they be implemented later?

Reply:
nvm, saw pr description, github buggy and won't let me delete this